rta/__init__.py

```diff
@@ -7,9 +7,15 @@
 import importlib
 import os
 
+from pathlib import Path
+
+from detection_rules.rule_loader import RuleCollection
```

Comment:
RTA is meant to be portable and in most cases will not have access to the accompanying detection_rules code